src/test/java/GetOrdersTests.java

```diff
@@ -0,0 +1,93 @@
+import ingredient.IngredientList;
+import io.qameta.allure.Step;
+import io.qameta.allure.junit4.DisplayName;
+import io.restassured.response.Response;
+import order.OrderRequest;
+import order.OrderResponse;
+import order.OrdersUser;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import user.UserCreds;
+import user.UserRequest;
+import user.UserResponse;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.apache.http.HttpStatus.SC_UNAUTHORIZED;
+import static org.junit.Assert.*;
+
+public class GetOrdersTests {
```

Comment:
java.lang.RuntimeException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "fat" (class ingredient.Ingredient), not marked as ignorable (11 known properties: "proteins", "carbohydrates", "calories", "image_large", "price", "_id", "name", "image_mobile", "type", "image", "__v"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 130] (through reference chain: ingredient.IngredientList["data"]->java.util.ArrayList[0]->ingredient.Ingredient["fat"])